package.json

```diff
@@ -86,4 +86,4 @@
         "webpack-dev-middleware": "^1.12.0",
         "webpack-glob-entries": "^1.0.1"
     }
-}
\ No newline at end of file
+}
```

Comment:
Probably should remove the new line to minimize changes to unrelated files.

Reply:
updated.